src/Backend/Core/Engine/DataGrid.php

```diff
@@ -113,11 +113,13 @@ public function addColumn(
         $image = null,
         $sequence = null
     ) {
+        // make sure we use a lowercased column everywhere (useful for checks, icons and more)
+        $name = mb_strtolower($name);
```

Comment:
You shouldn't do this as it can break css, we only need it lowercased for the icon check